packages/diagrams/backend/sirius-components-diagrams-layout/src/main/java/org/eclipse/sirius/components/diagrams/layout/ELKLayoutedDiagramProvider.java

```diff
@@ -285,4 +299,87 @@ private Position getAbsolutePosition(ElkNode node) {
         }
         return absolutePosition;
     }
+
+    private double getMaxPadding(Node node, ISiriusWebLayoutConfigurator layoutConfigurator) {
+        double maxPadding = 0d;
+        ElkPadding nodeLabelsPadding = layoutConfigurator.configureByType(node.getType()).getProperty(CoreOptions.NODE_LABELS_PADDING);
+        double leftPadding = 0d;
+        double rightPadding = 0d;
+        double topPadding = 0d;
+        double bottomPadding = 0d;
+
+        leftPadding = nodeLabelsPadding.getLeft();
+        rightPadding = nodeLabelsPadding.getRight();
+        topPadding = nodeLabelsPadding.getTop();
+        bottomPadding = nodeLabelsPadding.getBottom();
+        maxPadding = Double.max(Double.max(leftPadding, rightPadding), Double.max(topPadding, bottomPadding));
+
+        return maxPadding;
```

Comment:
Minor style issue, but why declare all these double here just to unconditionally assign them an actual value just below?
Also (but you know it), this is duplicate in two other places.

Reply:
Done.